tests/test_rpc_action.c

```diff
@@ -30,7 +30,7 @@
 
 #include "common.h"
 #include "sysrepo.h"
-#include "tests/config.h"
+#include "tests/common.h"
```

Comment:
The similar names of `common.h` and `tests/common.h` are a potential danger so to avoid future problems maybe rename to `tcommon.h` (the test one) or something.

Reply:
test_common.h seems best then?

Reply:
Sure.